hexrdgui/image_canvas.py

```diff
@@ -1034,10 +1037,39 @@ def finish_show_polar(self, iviewer):
                 }
                 self.axes_images.append(self.axis.imshow(**kwargs))
                 self.axis.axis('auto')
+
+                self.axis.yaxis.set_major_locator(AutoLocator())
+                self.axis.yaxis.set_minor_locator(AutoMinorLocator())
+
+                self.axis.xaxis.set_major_locator(AutoLocator())
+                self.axis.xaxis.set_minor_locator(AutoMinorLocator())
```

Comment:
I think we can use the `PolarXAxisTickLocator` to fix the major locator for when `Q` is the x-axis type. However, the minor locator will need to either be removed, or a custom minor locator will need to be made, for when `Q` is the x-axis type.

Reply:
We'll get it in this PR.

Reply:
It's done. Christos added a commit that fixed it.